auth_token/utils.py

```diff
@@ -1,34 +1,148 @@
 import binascii
-import os
 import random
 import re
+import string
 
+from datetime import timedelta
+from functools import reduce
+
+from operator import and_ as AND
+
+import import_string
+
+from django.core.exceptions import ImproperlyConfigured
 from django.conf import settings as django_settings
 from django.contrib.auth import _get_backends, load_backend
 from django.contrib.auth.models import AnonymousUser
 from django.contrib.auth.signals import user_logged_in, user_logged_out
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Q
 from django.middleware.csrf import rotate_token
-from django.utils.encoding import force_text
+from django.utils.crypto import salted_hmac
+from django.utils.timezone import now
+
+from .config import settings
+from .enums import AuthorizationRequestState, AuthorizationRequestResult
+from .signals import authorization_denied, authorization_granted, authorization_cancelled
 
-from auth_token.config import settings
 from ipware.ip import get_client_ip
 
 
+def _load_request_backend(path):
+    return import_string(path)
+
+
+def get_authorization_request_backends():
+    return [
+        (backend_path, _load_request_backend(backend_path))
+        for backend_path in settings.AUTHORIZATION_REQUEST_BACKENDS
+    ]
+
+
+def get_authorization_backend(path, raise_exception=False):
+    backends = dict(get_authorization_request_backends())
```

Comment:
Why isn't the dict created in `get_authorization_request_backends`, why is it a list of tuples?